packages/config/src/base.js

```diff
@@ -8,4 +10,17 @@ const getInitialBase = function ({
   return initialBase
 }
 
-module.exports = { getInitialBase }
+// Retrieve the final `base` directory used:
+//  - To load the second config file
```

Comment:
[dust] Please ignore if this is obvious and I'm just lacking context, but could we clarify in the comment what this second config file is?

Reply:
:+1: 
Done in 3489e7be71be8d776a9d459bb4a9858c03f2a92d